AdminUi/apps/admin_ui/lib/main.dart

```diff
@@ -68,13 +78,38 @@ class MainApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    const colorSchemeSeed = Color.fromARGB(255, 23, 66, 141);
-    return MaterialApp.router(
-      debugShowCheckedModeBanner: false,
-      themeMode: ThemeMode.light,
-      theme: ThemeData(colorSchemeSeed: colorSchemeSeed, useMaterial3: true),
-      darkTheme: ThemeData(brightness: Brightness.dark, colorSchemeSeed: colorSchemeSeed, useMaterial3: true),
-      routerConfig: _router,
+    return DynamicColorBuilder(
```

Comment:
as we do not want to harmonize our colors in code you can remove the dynamic color builder and just add the dark scheme from `color_schemes.dart`